### PR TITLE
[TM] Chp2-2  Update and center the chapter on first map loading and creation 

### PR DESCRIPTION
**Pull request by DelazJ**
With the exercise_data files modification, instructions needed some updates:
- roads layer is no more a shp file --> replaced in lesson with protected_areas.shp file (+screenshots)
- add details to the try yourself instructions
- introduces a `solution` folder next to exercise_data to avoid saving modified/new files (data and project) at the same place as original.

Also, it removes the geoPackage loading example, as it explores concepts not yet exposed at this stage. The lesson is thus recentered on first map loading and saving instructions.

PS: Actually the gpkg part is moved to another chapter (stay tuned!!!).

### CODE REVIEW

File: source/docs/training_manual/answers/answers.rst [hunk 1]
```diff
@@ -13,8 +13,12 @@ Answer Sheet
 |basic| *Preparation*
 ...............................................................................
 
-You should see a lot of lines, symbolizing roads. All these lines are in the
-vector layer that you just loaded to create a basic map.
+You should see in the main area of the dialog different shapes of different
```
Comment (havatv) [inline source/docs/training_manual/answers/answers.rst]:
```suggestion
In the main area of the dialog you should see many shapes with different
```

File: source/docs/training_manual/answers/answers.rst [hunk 2]
```diff
@@ -13,8 +13,12 @@ Answer Sheet
...
+colors. Each entity belongs to a layer you can identify by its color, in the
```
Comment (havatv) [inline source/docs/training_manual/answers/answers.rst]:
```suggestion
colors. Each entity belongs to a layer you can identify by its color in the
```

File: source/docs/training_manual/introduction/preparation.rst [hunk 3]
```diff
@@ -25,70 +25,53 @@ configured its installation.
...
-#. Look for and click the :guilabel:`Open Data Source Manager` button: |dataSourceManager|
+#. Look for and click the :guilabel:`Open Data Source Manager` button: |dataSourceManager|.
+   If you can't find the icon, check that the :guilabel:`Data Source Manager`
+   toolbar is enabled in the :menuselection:`View --> Toolbars` menu.
 
-   The dialog allows you to choose the data to load depending on the data type:
-   each tab on the left part of the dialog lists the available data types.
+   The :guilabel:`Data Source Manager` dialog allows you to choose the data to
+   load depending on the data type:
+   each tab on the left part of the dialog lists the main data types.
```
Comment (havatv) [inline source/docs/training_manual/introduction/preparation.rst]:
```suggestion
   the tabs in the left part of the dialog list the main data types.
```

File: source/docs/training_manual/introduction/preparation.rst [hunk 4]
```diff
@@ -25,70 +25,53 @@ configured its installation.
...
+.. _backlink-interface-preparation-1:
 
-|moderate| |FA| Load a layer from GeoPackage
+|basic| |TY|
 -------------------------------------------------------------------------------
 
-`GeoPackage <https://www.geopackage.org/>`_ is an open format for storing
-geospatial data. QGIS adds a lot of support to this new format that is slowly
-replacing the ESRI Shapefile format.
+Repeating the steps above add the :file:`places.shp` and :file:`rivers.shp`
```
Comment (havatv) [inline source/docs/training_manual/introduction/preparation.rst]:
```suggestion
Repeat the steps above to add the :file:`places.shp` and :file:`rivers.shp`
```

File: source/docs/training_manual/introduction/preparation.rst [hunk 5]
```diff
@@ -25,70 +25,53 @@ configured its installation.
...
+files from the same :file:`exercise_data/shapefile` folder to the map.
```
Comment (havatv) [inline source/docs/training_manual/introduction/preparation.rst]:
```suggestion
data sets from the same folder (:file:`exercise_data/shapefile`) to the map.
```

File: source/docs/training_manual/introduction/preparation.rst [hunk 6]
```diff
@@ -25,70 +25,53 @@ configured its installation.
...
+#. Save the map under a :file:`solution` folder next to :file:`exercise_data`
+   and call it :file:`basic_map.qgs`.
 
-:ref:`Check your results <interface-preparation-1>`
-
-.. _load_geopackage:
+.. _backlink-interface-preparation-1:
 
-|moderate| |FA| Load a layer from GeoPackage
+|basic| |TY|
```
Comment (havatv) [inline source/docs/training_manual/introduction/preparation.rst]: Travis complains: `Undefined substitution referenced: "TY".`

File: source/docs/training_manual/answers/answers.rst [hunk 7]
```diff
@@ -13,8 +13,12 @@ Answer Sheet
 |basic| *Preparation*
 ...............................................................................
 
-You should see a lot of lines, symbolizing roads. All these lines are in the
-vector layer that you just loaded to create a basic map.
+In the main area of the dialog you should see many shapes with different
+colors. Each entity belongs to a layer you can identify by its color in the
```
Comment (tomchadwin) [inline source/docs/training_manual/answers/answers.rst]:
```suggestion
colors. Each shape belongs to a layer you can identify by its color in the
```

File: source/docs/training_manual/introduction/preparation.rst [hunk 8]
```diff
@@ -25,70 +25,53 @@ configured its installation.
...
 Let's get started right away!
 
-.. _backlink-interface-preparation-1:
-
 |basic| |FA| Prepare a map
 -------------------------------------------------------------------------------
 
 #. Open QGIS. You will have a new, blank map.
-#. Look for and click the :guilabel:`Open Data Source Manager` button: |dataSourceManager|
+#. Look for and click the :guilabel:`Open Data Source Manager` button: |dataSourceManager|.
```
Comment (tomchadwin) [inline source/docs/training_manual/introduction/preparation.rst]:
```suggestion
#. Click the :guilabel:`Open Data Source Manager` button: |dataSourceManager|.
```

File: source/docs/training_manual/introduction/preparation.rst [hunk 9]
```diff
@@ -25,70 +25,53 @@ configured its installation.
...
+   If you can't find the icon, check that the :guilabel:`Data Source Manager`
+   toolbar is enabled in the :menuselection:`View --> Toolbars` menu.
 
-   The dialog allows you to choose the data to load depending on the data type:
-   each tab on the left part of the dialog lists the available data types.
+   The :guilabel:`Data Source Manager` dialog allows you to choose the data to
+   load depending on the data type:
+   the tabs in the left part of the dialog list the main data types.
```
Comment (tomchadwin) [inline source/docs/training_manual/introduction/preparation.rst]:
```suggestion
   the tabs in the left-hand section of the dialog list the main data types.
```

File: source/docs/training_manual/introduction/preparation.rst [hunk 10]
```diff
@@ -25,70 +25,53 @@ configured its installation.
...
 
-|moderate| |FA| Load a layer from GeoPackage
+|basic| |TY|
 -------------------------------------------------------------------------------
 
-`GeoPackage <https://www.geopackage.org/>`_ is an open format for storing
-geospatial data. QGIS adds a lot of support to this new format that is slowly
-replacing the ESRI Shapefile format.
+Repeat the steps above to add the :file:`places.shp` and :file:`rivers.shp`
+data sets from the same folder (:file:`exercise_data/shapefile`) to the map.
```
Comment (tomchadwin) [inline source/docs/training_manual/introduction/preparation.rst]: datasets/data sets?

File: source/docs/training_manual/introduction/preparation.rst [hunk 11]
```diff
@@ -98,7 +81,7 @@ You've learned how to add a layer and create a basic map!
 |WN|
 -------------------------------------------------------------------------------
 
-Now you're familiar with the function of the :guilabel:`Open Data Source Manager`
+Now you're familiar with the basic function of the :guilabel:`Open Data Source Manager`
 button, but what about all the others? How does this interface work? Before we
 go on with the more involved stuff, let's first take a good look at the general
```
Comment (tomchadwin) [inline source/docs/training_manual/introduction/preparation.rst]:
```suggestion
go on, let's first take a look at the
```

**Comment by havatv** [comment]
Travis: "training_manual/answers/answers.rst:61:undefined label: load_geopackage"

**Comment by DelazJ** [comment]
> Travis: "training_manual/answers/answers.rst:61:undefined label: load_geopackage"

Should be fixed with a rebase after #3300 is merged (https://github.com/qgis/QGIS-Documentation/pull/3300/files#diff-17da272b98fc65b607132a217a74c434L54)

**Pull request merged**